.circleci/config.yml

```diff
@@ -326,4 +326,9 @@ commands:
             if git log -1 "$CIRCLE_SHA1" | grep -q '\[do not deploy\]'; then
                 echo "Skipping remaining steps in this job: deployment was disabled for this commit."
                 circleci-agent step halt
+
```

Comment:
Does it make sense to keep the `circleci-agent step halt` call here or should we remove that?

Reply:
No worries. Hmm, looks like we will need a follow-up for this. It still asked me to approve when I merged this PR to main. Investigating.

Reply:
Hi @ncloudioj! 👋🏻 

Did you have a chance to look into this?

Reply:
@hackebrot Just wanted to circle back on this, turns out that CircleCI was also not happy with using a personal API token here, a "permission denied" error was thrown when trying to cancel the rest of build tasks on CircleCI. Looks like we need to set up another API token just CI for this. Will reach out to SRE folks for some help.

Reply:
Thanks, @ncloudioj! I've been following your conversation with @dlactin on Slack. 👍🏻